src/components/DepositSection.tsx

```diff
@@ -60,7 +71,7 @@ export const DepositSection: React.FC<unknown> = () => {
   return (
     <div className="sm:grid sm:grid-cols-1 sm:items-start sm:gap-1.5 sm:py-3">
       <label htmlFor="description" className="text-sm font-medium text-blue">
-        Deposit
+        Deposit {scaledMinDeposit ? scaledMinDeposit[1] : ""}
```

Comment:
on a local chain, the deposit was something like 0.000001BLD. This UI gave a corresponding default, but wouldn't accept it. It said something like "the closest values are 0 and 1". Not a big deal.